README.md

```diff
@@ -35,6 +35,31 @@ See [terraform-getting-started](https://developer.hashicorp.com/terraform/tutori
 
 See [getting-started-install](https://docs.aws.amazon.com/cli/latest/userguide/getting-started-install.html)
 
+
+
+### Setup a local SM2A development environment
+
+1. Initialize the metadata db (only needed once)
+
+```shell
+make sm2a-local-init
```

Comment:
I wonder if this needs to be a make command if it's only ever run once?

Could it just live in the README?

I'm not tied to either approach, but I would assume most make commands would be used frequently

Reply:
It was my motivation to have a makefile here, without it, the user would have to type three long command lines instead of simply running 'make init`

Reply:
I guess I'd assume most would just copy paste 😛

Reply:
Yes, but I cant count how many time someone missed `cp .env_example .env` while it is stated in README.
. Also, I was mistaken earlier the `init` needs to be run initially to set up the database and create the user. It will need to be run again if `make clean` is executed. I have updated the README to reflect this.